src/__test__/connection/reconnect.test.ts

```diff
@@ -26,7 +26,9 @@ describe("reconnect", () => {
     // make successful append to connect to node
     const firstAppend = await client.appendToStream(
       "my_stream",
-      jsonEvent({ type: "first-append", data: { message: "test" } })
+      jsonEvent({ type: "first-append", data: { message: "test" } }),
+      // batch append triggers reconnect as soon as stream drops, so we need to force regular append
+      { credentials: { username: "admin", password: "changeit" } }
```

Comment:
**QUESTION:** If someone uses explicit ACL params in append, then batch won't be used?

Reply:
As discussed:
- no explicit setting
- rewrite test not to use append, to avoid workaround

Reply:
👍